internal/cmd/commands/workerscmd/certificate-authority.go

```diff
@@ -0,0 +1,220 @@
+package workerscmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/boundary/api"
+	"github.com/hashicorp/boundary/api/workers"
+	"github.com/hashicorp/boundary/internal/cmd/base"
+	"github.com/hashicorp/boundary/internal/cmd/common"
+	"github.com/hashicorp/go-secure-stdlib/strutil"
+	"github.com/mitchellh/cli"
+	"github.com/mitchellh/go-wordwrap"
+	"github.com/posener/complete"
+)
+
+var (
+	_ cli.Command             = (*WorkerCACommand)(nil)
+	_ cli.CommandAutocomplete = (*WorkerCACommand)(nil)
+)
+
+type WorkerCACommand struct {
+	*base.Command
+
+	Func string
+}
+
+func (c *WorkerCACommand) Synopsis() string {
+	switch c.Func {
+	case "read":
+		return wordwrap.WrapString("Read the certificate authority used to authorize Boundary workers", base.TermWidth)
+	case "reinitialize":
+		return wordwrap.WrapString("Reinitialize the certificate authority used to authorize Boundary workers", base.TermWidth)
+	}
+	return wordwrap.WrapString("Manage the certificate authority used to authorize Boundary workers", base.TermWidth)
+}
+
+var flagsCertificateAuthority = map[string][]string{
+	"reinitialize": {"scope-id"},
+	"read":         {"scope-id"},
+}
+
+func (c *WorkerCACommand) Help() string {
+	initFlags()
+	switch c.Func {
+	case "read":
+		return base.WrapForHelpText([]string{
+			"Usage: boundary workers certificate-authority read",
+			"",
+			"  Read the certificate authority used to authorize Boundary workers:",
+			"",
+			`    $ boundary workers certificate-authority read`,
+			"",
+			"",
+		}) + c.Flags().Help()
+	case "reinitialize":
+		return base.WrapForHelpText([]string{
+			"Usage: boundary workers certificate-authority reinitialize",
+			"",
+			"  Reinitialize the certificate authority used to authorize Boundary workers:",
+			"",
+			`    $ boundary workers certificate-authority reinitialize`,
+			"",
+			"",
+		}) + c.Flags().Help()
+	}
+	return base.WrapForHelpText([]string{
+		"Usage: boundary workers certificate-authority [options]",
```

Comment:
See internal/cmd/common/help.go  line 44: the general pattern here is to say something high level, then have an example command. It's a subtle difference but one that makes it clear that the user didn't run the command the wrong way just missing `read`, that `read` is simply _one_ example of something that can be done. Then the "Please see the {{hyphentype}}s subcommand..." bit makes it clear to look at the available subcommands.

Reply:
Thanks, I've updated this!